blueprint/core/blueprint_View.cpp

```diff
@@ -191,7 +191,7 @@ namespace blueprint
             auto borderBounds = getLocalBounds().toFloat().reduced(borderWidth * 0.5f);
             auto width  = borderBounds.getWidth();
             auto height = borderBounds.getHeight();
-            auto minLength = std::min(width, height);
+            auto minLength = juce::jmin(width, height);
```

Comment:
Why this one?

Reply:
I noticed that `min` and `max` are defined, and conflicted with `std::min`. Rather terrible library conflict, but a rather normal one.

Reply:
Well, I don't really understand it, but I see now that this fixes a build error on windows! https://github.com/nick-thompson/blueprint/pull/193/files

I'll go ahead and merge this PR here shortly as clearly it will unblock windows users at the same time! And then I'll follow up to merge my CI PR

Reply:
It's cross-platform of course. The issue is that the `Windows.h` header _defines_ `min` and `max`, and some of the underlying math headers in the MSVC toolchain do so as well.